lib/tag/tag.js

```diff
@@ -28,7 +28,9 @@ function Tag(impl, conf, innerHTML) {
 
   // grab attributes
   each(root.attributes, function(el) {
-    attr[el.name] = el.value
+    var val = el.value
+    // remember attributes with expressions only
+    if (/\{.*\}/.test(val)) attr[el.name] = val
```

Comment:
You should use `brackets()` for expressions instead of `{}` since it is configurable.